Add example /usr/bin/env python as Python interpreter (#280)
ansible/ansible-documentation: docs/docsite/rst/reference_appendices/python_3_support.rst

    $ ansible localhost-py3 -m ping
    $ ansible-playbook sample-playbook.yml

> * If you want to use the first Python found on ``PATH`` or Python interpreter path can't be known in advance, you can use ``/usr/bin/env python`` such as:

Reviewer: ```suggestion
* To use the first Python found on ``PATH`` or if the Python interpreter path is not known in advance, you can use ``/usr/bin/env python`` such as:
```
